Add B01

TrueFinch opened the pull request:
Add B01 tasks. Add to GenBase.py class DirectInput for questions of type 'DirectInput'

https://www.notion.so/EGE-B01-4224059287294ab987bae4ced7627c04

TrueFinch committed:
Add class for DirectInput type questions

https://www.notion.so/EGE-B01-4224059287294ab987bae4ced7627c04

TrueFinch committed:
Add B01 tasks

https://www.notion.so/EGE-B01-4224059287294ab987bae4ced7627c04

TrueFinch committed:
Add docstrings to classes Record2 and Direct

https://www.notion.so/EGE-B01-4224059287294ab987bae4ced7627c04

ToriRori commented on EGE/GenBase.py:
```diff
@@ -54,3 +54,19 @@ def post_process(self):
+        if re.search(self.accept, self.correct):
+            return
+        # TODO: ask ask if message is correct
+        raise EGEError(f"Correct answer is not acceptable in {self}: {self.correct}")
```
Не хватает пустой строки в конце

TrueFinch replied:
Done

TrueFinch committed:
Add B01 task to gen.py

https://www.notion.so/EGE-B01-4224059287294ab987bae4ced7627c04 — also fix error with no variants field in DirectInput class (TODO: maybe this is not the best solution, idk)

TrueFinch committed:
Add blank line at file's end

https://www.notion.so/EGE-B01-4224059287294ab987bae4ced7627c04 — according to https://github.com/klenin/EGEpy/pull/15#discussion_r780643705
